sdk/bom/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
We have the opentelemetry bom, is there a benefit in having another bom @jkwatson ?

Reply:
I actually have no idea. The last I heard, the bom wasn't being properly published. Not a bom expert here.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/1413 for reference

Reply:
Yeah - I use the bom from the snapshots repo right now and it's fine, we'll need to figure out what's up with maven central, but I wouldn't add another bom which is just a subset of it.

Reply:
On #1460 @bogdandrutu had requested a bom for the SDK pieces.

If there's agreement it's not needed, I can take it out. I'm fine with whatever the majority want

Reply:
cool. if the existing bom encompasses this already, then I think we don't need another. 👍

Reply:
I removed the SDK BOM